roles/cfg_openwrt/templates/corerouter/config/dhcp.j2

```diff
@@ -13,10 +13,20 @@ config dnsmasq
 	option noping '{{ dhcp_no_ping|int if dhcp_no_ping is defined else 1 }}'
 	option quietdhcp '1'
 	option cachesize '{{ dns_cache_size | default(10000) }}'
+ 	list server '/olsr/'
 {% for _dns_server in dns_servers %}
         list server '{{ _dns_server }}'
 {% endfor %}
 
+{% for network in networks | selectattr('role', 'equalto', 'mgmt') %}
+  {% for host, ip_num in network['assignments'].items() %}
+config domain '{{ host | replace('-', '_') }}_olsr'
+	option name '{{ host }}.olsr'
+	option ip '{{ network['prefix'] | ansible.utils.ipaddr(ip_num) | ansible.utils.ipaddr('address') }}'
+
+  {% endfor %}
+{% endfor %}
```

Comment:
Do I understand this right, that this will include a list of all bbb-configs-known nodes to the dnsmasq config?

Reply:
Only all mgmt assignments of the current location, as in `networks` in networks.yml. It's basically the same listing that we do in olsrd.conf to configure the nameservice plugin.

Making this explicit to dnsmasq means we don't need to wait precious minutes for the domain names to become available.